src/lib.rs

```diff
@@ -119,6 +119,26 @@ impl Device {
         }
     }
 
+    pub fn usb_close(mut self) -> Result<Builder> {
```

Comment:
Do we have an actual use case for this, or do you do this just as it was in #2? I'd expect that allocating memory for the context is cheaper than actually opening the device, so the optimization is minimal.